src/main/java/com/minecolonies/client/render/EmptyTileEntitySpecialRenderer.java

```diff
@@ -23,6 +30,12 @@
     @Override
     public void renderTileEntityAt(TileEntityColonyBuilding tileEntity, double x, double y, double z, float partialTicks, int destroyStage)
     {
+        if(tileEntity.getBuilding() instanceof BuildingTownHall)
+        {
+            String text = "Blablablah";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'text' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)